rocon_interactions/src/rocon_interactions/manager.py

```diff
@@ -71,8 +71,9 @@ def __init__(self):
         self._parameters = self._setup_parameters()  # important to come first since we use self._parameters['pairing'] everywhere
         self._publishers = self._setup_publishers()  # important to come early, so we can publish is_pairing below
         self._rapp_handler = None
+        if self._parameters['pairing'] or self._parameters['rapp_manager']:
```

Comment:
Probably good enough to keep it simple with one parameter for now as we don't have a use case to split the functionality yet (and probably not likely to).